installer/scripts/omsadmin.sh

```diff
@@ -353,6 +353,15 @@ onboard_scom()
     #Always register SCOM as secondary Workspace
     echo "SCOM Workspace" > $CONF_DIR/.multihoming_marker
     configure_logrotate
+
+    #Open port 1270 if not already open
+    IS_PORT_OPEN=`cat $OMI_CONF_FILE | grep httpsport | grep 1270 | wc -l`
```

Comment:
No, not exactly. The typical line here would be `httpsport=0,1270` due to an artifact on how `omiconfigeditor` works. The 0 would be ignored.

This is actually harder than you might think. What if port 11270 is exposed. Looking for 1270 would work, but would actually be wrong. And `omiconfigeditor` isn't going to help you because it only has mechanisms to change a config file, not to help read it and parse it. This is trivial in python, but we don't have that readily available here.

Our existing scripting is wrong in this regard. And it's hard to do in bash, not to mention cross platform (UNIX, which we should get right).  Things to be concerned about:

1. Is httpsport commented out? If so, ignore it.
2. Port 1270 is different from 11270, and we should know that.
3. The line can contain `=1270`, `=0,1270`, or `=0, 1270`, and we should recognize that.

In retrospect, if we wanted to fix this dependably, we should likely modify `omiconfigeditor` to have a query command (like `-q 1270 httpsport`) that would return `true` or `false` based on what you are querying for. It already knows how to deal with commented out entries, and comma-separated entries. It should be easy there.

Either that, or do what Laura said (actually `grep httpsport $OMI_CONF_FILE | grep 1270 > /dev/null 2> /dev/null`), but know that that is broken in certain cases.

But I think `omiconfigeditor` is the "right" way to do this, and can then be used in lots of other places (including UNIX) trivially.